Using-browsec-for-securing-your-connection-to-fcc.md

```diff
@@ -0,0 +1,63 @@
+# Using Browsec for Securing your Connection to FreeCodeCamp.com
+
+## Why do I need an plugin?
+Lately some of the campers started facing a strange problem, namely content blocking by some Internet Service Providers (ISP), leading to the [FreeCodeCamp.com](http://freecodecamp.com) (FCC) website being broken sometimes.
+
+This came soon to light with some of the issues logged by campers :
+- [\#5999 : Indian ISP's injecting Ads breaks site](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/5999)
+- [\#6122 : Serious Problem using the FCC IDE, console showing error...](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6122)
+- [\#6381 : Unable to view the Code editor in the browser](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6381)
+
+## TL;DR : What's Wrong, again?
+
+Well for some campers this is a classic case of man in the middle attack, for instance in the issue \#5999 the ISP some of the campers from India are using is deliberately, injecting Ads into the user's visited websites that's causing the issue.
```

Comment:
Add Wikipedia Link on `man in the middle attack`
https://en.wikipedia.org/wiki/Man-in-the-middle_attack

You may want to explain that they are modifying websites as they are transmitted.  In some cases this is to serve ads, and in other cases . . .  etc.